tests/xtimer_longterm/main.c

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       xtimer_msg test application
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @}
+ */
+
+#include <stdio.h>
+#include <time.h>
+
+#include "xtimer.h"
+#include "thread.h"
+#include "msg.h"
+
+
+/* some internally used msg types */
+#define MSG_LONG                (0xcafe)
+#define MSG_MID                 (0xe5e1)
+#define MSG_TICK                (0xaffe)
+
+/* define sleep and timeout intervals */
+#define MIN_TO_TICKS(min)       (XTIMER_USEC_TO_TICKS(min * 60 * 1000 * 1000))
```

Comment:
You should change the order and make the first constant "long unsigned" (LU), otherwise this breaks for 16bit platforms.